### PR TITLE
Gin i18n middleware

### PR DESCRIPTION
**Pull request by bos-hieu**
- This is a gin i18n middleware that wrapper [go-i18n](https://github.com/nicksnyder/go-i18n)
- @thinkerou Please review my PR! Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,162 @@
...
+Download and install it:
+
+```sh
+go get github.com/bos-hieu/gin-i18n
+```
+
+Import it in your code:
+
+```go
+import gini18n "github.com/bos-hieu/gin-i18n"
```
[the remark this answers is not in the document]
Comment (bos-hieu) [reply README.md]: I just updated. Please take a look @appleboy. Thanks!

File: README.md [hunk 2]
```diff
@@ -1 +1,162 @@
...
+
+```go
+package main
+
+import (
+	gini18n "github.com/bos-hieu/gin-i18n"
+	"github.com/gin-gonic/gin"
+	"github.com/nicksnyder/go-i18n/v2/i18n"
+	"log"
+	"net/http"
```
[the remark this answers is not in the document]
Comment (bos-hieu) [reply README.md:30]: I just updated. Please take a look @thinkerou. Thanks!

File: README.md [hunk 3]
```diff
@@ -1 +1,166 @@
-# i18n
\ No newline at end of file
+# i18n
+
+[![Run Tests](https://github.com/bos-hieu/gin-i18n/actions/workflows/go.yml/badge.svg)](https://github.com/bos-hieu/gin-i18n/actions/workflows/go.yml)
```
Comment (appleboy) [inline README.md:3]: update bos-hieu

File: constant.go [hunk 4]
```diff
@@ -0,0 +1,29 @@
+package gini18n
```
Comment (appleboy) [inline constant.go:1]: change package name to `i18n`

**Review by appleboy — CHANGES_REQUESTED** [review]

File: constant.go (hunk 4 above)
Comment (bos-hieu) [reply constant.go]: @appleboy I updated code from your comment. Please review these, thanks!

File: .github/workflows/go.yml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+      - name: Setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.16'
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.42.1
```
Comment (appleboy) [inline .github/workflows/go.yml:24]: v1.43.0

**Review by appleboy — CHANGES_REQUESTED** [review]

File: .github/workflows/go.yml (hunk 5 above)
Comment (bos-hieu) [reply .github/workflows/go.yml]: @appleboy I just updated the version. Thanks!

**Review by appleboy — APPROVED** [review]
LGTM

**Comment by appleboy** [comment]
Wait for @thinkerou's review.

**Comment by bos-hieu** [comment]
@thinkerou Thank you in advance for your feedback.

**Review by appleboy — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -1 +1,166 @@
...
+[![GoDoc](https://godoc.org/github.com/gin-contrib/gin-i18n?status.svg)](https://godoc.org/github.com/gin-contrib/gin-i18n)
+[![Go Report Card](https://goreportcard.com/badge/github.com/gin-contrib/gin-i18n)](https://goreportcard.com/report/github.com/gin-contrib/gin-i18n)
+
+## Usage
+
+Download and install it:
+
+```sh
+go get github.com/gin-contrib/gin-i18n
+```
```
Comment (thinkerou) [inline README.md:15]: `github.com/gin-contrib/i18n`?

Comment (bos-hieu) [reply README.md:15]: @thinkerou I updated this. Thank so much for your feedback!

**Review by thinkerou — APPROVED** [review]
lgtm

**Pull request merged**